docs/index.rst

```diff
@@ -12,25 +12,33 @@ Welcome to the hipSPARSELt docs home page! To learn more, see :ref:`what-is-hips
 
 Our documentation is structured as follows:
 
-.. grid:: 1 2 2 2
+.. grid:: 2
   :gutter: 3
 
-  .. grid-item-card:: Reference
+  .. grid-item-card:: Install
 
-    * :ref:`Supported functions <supported-functions>`
-    * :ref:`Data type support <data-type-support>`
-    * :ref:`Device & stream management <device-stream-manage>`
-    * :ref:`Storage formats <storage-format>`
-    * :ref:`Porting from CUDA <porting>`
+    * :doc:`Quick-start <./install/quick-start>`
+    * :doc:`hipSPARSELt installation <./install/linux>`
+
+  .. grid-item-card:: API reference
```

Comment:
this list is in a different order than in the toc

Reply:
Here's what i have:

.. grid:: 2
  :gutter: 3

  .. grid-item-card:: Install

    * :doc:`Quick-start <./install/quick-start>`
    * :doc:`Installation guide <./install/linux>`

  .. grid-item-card:: API reference

    * :doc:`Supported functions <./reference/supported-functions>` 
    * :doc:`Data type support <./conceptual/data-type-support>`
    * :doc:`API library <../doxygen/html/index>`

  .. grid-item-card:: How to

    * :doc:`Port from CUDA to hipSPARSELt <./how to/porting>`

  .. grid-item-card:: Conceptual

     * :doc:`HIP device and stream management <./conceptual/device-stream-manage>`
     * :doc:`Storage formats <./conceptual/storage-format>`

Reply:
yes, I can see what you have in the file. The order of the API ref content differs from the order in the TOC.

Reply:
> yes, I can see what you have in the file. The order of the API ref content differs from the order in the TOC.

Same order in the .yml file. Rebuilt just now. 

- caption: Conceptual
  entries:   
    - file: conceptual/device-stream-manage.rst
      title: HIP device & stream management
    - file: conceptual/storage-format.rst
      title: Storage formats

- caption: How to
  entries:    
    - file: how to/porting
      title: Port from CUDA to hipSPARSELt

- caption: API Reference
  entries:   
    - file: reference/supported-functions.rst
      title: Supported functions   
    - file: conceptual/data-type-support.rst
      title: Data type support
    - file: doxygen/html/index
      title: API library

Reply:
> > yes, I can see what you have in the file. The order of the API ref content differs from the order in the TOC.
> 
> Same order in the .yml file. Rebuilt again.
> 
> * caption: Conceptual
>   entries:
>   
>   * file: conceptual/device-stream-manage.rst
>     title: HIP device & stream management
>   * file: conceptual/storage-format.rst
>     title: Storage formats
> * caption: How to
>   entries:
>   
>   * file: how to/porting
>     title: Port from CUDA to hipSPARSELt
> * caption: API Reference
>   entries:
>   
>   * file: reference/supported-functions.rst
>     title: Supported functions
>   * file: conceptual/data-type-support.rst
>     title: Data type support
>   * file: doxygen/html/index
>     title: API library

Reply:
Why is data type support in the conceptual folder? Shouldn't it be in the reference folder? Also, it's no longer showing up in the TOC and the link is broken on the index page.

Reply:
It's in the reference folder,

https://github.com/Rmalavally/hipSPARSELt/tree/hipsparseltdocs_reorg/docs/reference

Reply:
Only two files in the conceptual folder

https://github.com/Rmalavally/hipSPARSELt/tree/hipsparseltdocs_reorg/docs/conceptual

Reply:
Data type support.rst was in both directories. Removed it from conceptual.

Reply:
(you don't need to paste file contents into comments--I can easily see them within the files themselves)